Gemfile

```diff
@@ -1,22 +1,24 @@
 source 'https://rubygems.org'
 
-gem 'celluloid'
-gem 'celluloid-io'
-gem 'http'
+ruby RUBY_VERSION
```

Comment:
@olleolleolle @tarcieri sorry to harp on this, but this [Gemfile directive](http://bundler.io/v1.15/man/gemfile.5.html#RUBY) is intended to specify a version of ruby the application requires. I don't think it's valid to use it here like this. thoughts?